docs/fsharp/language-reference/parameters-and-arguments.md

```diff
@@ -117,6 +119,19 @@ The output is as follows.
 Baud Rate: 9600 Duplex: Full Parity: false
 Baud Rate: 4800 Duplex: Half Parity: false
 Baud Rate: 300 Duplex: Half Parity: true
+Baud Rate: 9600 Duplex: Full Parity: false
+Baud Rate: 9600 Duplex: Full Parity: false
+Baud Rate: 4800 Duplex: Half Parity: false
+```
+
+Since F# 4.1 you can use the attributes `[<Optional; DefaultParameterValue<(...)>]` in F# for C# and VB interop, so that C#/VB callers will see an argument as optional. This is equivalent to defining the argument as optional in C# as in `MyMethod(int i = 3)`.
```

Comment:
Given that F# 4.1 was released in early 2017, I think we can drop the "Since F# 4.1" and just say, "You can...". It's probably also worth mentioning that the value passed into the constructor of `DefaultParameterValue` needs to match the type of the argument to the method.